docs(issue-views): Overhaul Issue Views docs for new UI (#13762)
getsentry/sentry-docs: docs/product/issues/issue-views/index.mdx

---
title: Issue Views
sidebar_order: 9
description: "Learn how to create, customize, and share your Issues experience in Sentry."
---

Issue views let you customize and share your issue stream, so you and your teammates can quickly see the issues that are most relevant to you.

<Alert>
  This feature is only available if your organization has enabled [early adopter features](/organization/early-adopter-features/). Early adopter features are still in-progress and may have bugs. We recognize the irony. If you’re interested in participating, enable early adopter features in [organization settings](https://sentry.io/orgredirect/organizations/:orgslug/settings/organization).
</Alert>

## Creating Your First View

To create your first custom issue view, navigate to the [**All Views**](https://sentry.io/orgredirect/organizations/:orgslug/issues/views/) page and click the **`+ Create View`** button on the top right.

![All Views Page](./img/issue_views_all_views_create_view.png)

This will create a new starred view, which you can than update with the filters of your choosing. Some common use cases for views include:

> * **Prioritized:** A view containing only issues with high or medium priority

Reviewer: IMO listing these here isn't necessary

![Unsaved Changes Indicator](./img/issue_views_unsaved_changes.png)

To persist these changes to your view, simply click on the purple `Save` button to the right of the search and sort inputs in the Issues page. You may also create a new view with these changes  by clicking `Save as new view`, or you can reset the view back to its original filters by clicking `Reset`.

![Save Button](./img/issue_views_save_button.png)

### Reordering Views

You can reorder the starred views in the navigation by hovering over them and clicking and dragging the handle to the desirable position.

### Renaming Views

To rename a view, navigate to that view's page and hover over the page header, which will reveal a pencil icon. Click the pencil icon to edit the view's name.

![Renaming](./img/issue_views_renaming.png)

You can also do this by navigating to the `All Views` page and clicking on the view's ellipsis menu and selecting `Rename`.

## Sharing Views

### Browsing and Starring Your Organization's Views

> All Issue Views are visible to all members of your organization, and you can browse your organization's views by navigating to the [**All Views**](https://sentry.io/orgredirect/organizations/:orgslug/issues/views/) page. Keep in mind that some views may filter on projects that you may not have access to. Those views will be inaccessible if open membership is not enabled.

Reviewer: Are the views inaccessible? I thought they were visible, but the search may not work

Author: ah yeah I meant that the list of issues would be inaccessible, but I can see why that would be confusing. Will update that wording

> All Issue Views are visible to all members of your organization, and you can browse your organization's views by navigating to the [**All Views**](https://sentry.io/orgredirect/organizations/:orgslug/issues/views/) page. Keep in mind that some views may filter on projects that you may not have access to. Those views will be inaccessible if open membership is not enabled.

Author: ```suggestion
All Issue Views are visible to all members of your organization, and you can browse your organization's views by navigating to the [**All Views**](https://sentry.io/orgredirect/organizations/:orgslug/issues/views/) page. Keep in mind that some views may filter on projects that you may not have access to. You will not be able to see the issues in these views if open membership is not enabled.
```

![All Views](./img/issue_views_all_views.png)

> Here, you can search and sort for specific views and star any views that you want to see in your sidebar.

Reviewer: nit: "sort for" doesn't seem grammatically correct

```suggestion
Here, you can search for specific views and star any that you want to see in the sidebar.
```
